### PR TITLE
Add workflow to run benchmarks

### PR DESCRIPTION
**Pull request by pioug**
Hello, I just found out colorette and was curious about how it performs. So I had to clone the repo to run the benchmark.

Would you mind if the repo runs regularly the benchmarks via GH actions? So anyone wouls be able to check the benchmarks results through the logs. The readme could link to the logs of this new workflow (for example: https://github.com/jorgebucaran/colorette/actions/workflows/bench.yml).

Data will be relatively up to date without requiring extra effort (no need to include the results in the repo). The results should be fairly consistent too since it runs on the CI server.

Here an example from my fork: https://github.com/pioug/colorette/runs/3575179792?check_suite_focus=true#step:4:15

### CODE REVIEW

File: .github/workflows/bench.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+jobs:
+  bench:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 16
+      - name: Benchmarks
+        run: npm --prefix bench start
```
Comment (jorgebucaran) [inline .github/workflows/bench.yml:12]: Should be `FORCE_COLOR= npm --prefix bench start` to make sure everyone is actually doing any work.

**Review by jorgebucaran — CHANGES_REQUESTED** [review]

**Comment by jorgebucaran** [comment]
Cool, thank you! 👍

**Comment by jorgebucaran** [comment]
@pioug Left you a comment.

**Comment by jorgebucaran** [comment]
This is awesome, thank you! Can't wait to merge this.

File: .github/workflows/bench.yml (hunk 1 above)
Comment (pioug) [reply .github/workflows/bench.yml]: Oh thanks, that explains the weird results 😅 It looks better now https://github.com/pioug/colorette/runs/3576204006?check_suite_focus=true#step:4:15

```
colorette × 502,111 ops/sec
chalk × 342,441 ops/sec
kleur × 372,220 ops/sec
colors × 170,611 ops/sec
ansi-colors × 213,404 ops/sec
```

**Comment by jorgebucaran** [comment]
@pioug Do you have any tips for directly linking to the benchmark results in the CI? 🤔

**Pull request merged**